rollbar_flutter/example/ios/Podfile.lock

```diff
@@ -8,20 +8,20 @@ PODS:
   - rollbar_flutter (0.0.1):
```

Comment:
I am not sure we need to have this file under git control. Probably can be added to .gitignore. I think it gets generated by Cocopods.

Reply:
Whether to keep dependency versioning lock files under version control is a decades long debate.

I think that, in the case of a Flutter project, I'd like to keep the versions locked only because it gives us more control over what gets installed and what doesn't.